config/locales/fil/simple_form.fil.yml

```diff
@@ -0,0 +1,55 @@
+# See this documentation: https://github.com/plataformatec/simple_form#i18n
+# And this StackOverflow answer: https://stackoverflow.com/questions/35906089/changing-simple-form-submit-button-i18n-text
+
+en:
```

Comment:
We can change this line from "en:" to "fil:" (otherwise it will conflict with the en" locale file.)